examples/CMakeLists.txt

```diff
@@ -3,6 +3,15 @@
 #    - Slaven Peles <peless@ornl.gov>
 #]]
 
+set(GRIDKIT_INSTALL_EXAMPLES_ROOT "share/gridkit/examples")
+
+macro(gridkit_example_install_path out_var)
+    cmake_path(RELATIVE_PATH CMAKE_CURRENT_SOURCE_DIR
+        BASE_DIRECTORY "${CMAKE_SOURCE_DIR}/examples"
+        OUTPUT_VARIABLE _rel_path)
+    set(${out_var} "${GRIDKIT_INSTALL_EXAMPLES_ROOT}/${_rel_path}")
+endmacro()
```

Comment:
- Do we need a macro for this? It is a one-liner, so macro looks like an overkill.
- Do we need to have literal path? What if we drop the quotation marks around examples install dir?